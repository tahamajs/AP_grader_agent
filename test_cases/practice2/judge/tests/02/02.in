enter 10
reserve_table 30 1
enter 10
reserve_table 3 6
enter 10
exit 30
enter 10
reserve_table 7 6
enter 10
reserve_table 9 6
enter 10
reserve_table 50 6
enter 10
exit 9
enter 10
exit 3
enter 10
enter 48
