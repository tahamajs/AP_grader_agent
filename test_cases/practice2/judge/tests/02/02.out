Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Mike sits at table 1
Table 2: 4 0
Table 6: 3 0
Table 1: 3 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Parsa sits at table 6
Table 6: 2 0
Table 2: 4 0
Table 5: 3 0
Table 1: 3 0
Table 4: 2 0
Table 3: 2 0
Mike exits!
Table 2: 4 0
Table 6: 2 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Mahdi sits at table 6
Table 2: 4 0
Table 6: 1 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Sadegh sits at table 6
Table 2: 4 0
Table 6: 0 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Sobhan enters the waiting queue of table 6
Table 2: 4 0
Table 6: 0 1
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Sadegh exits!
Table 2: 4 0
Table 6: 0 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Parsa exits!
Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 1 0
Table 3: 2 0
Table 4: 2 0
Table 2: 4 0
Table 6: 1 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
